include/oneapi/tbb/detail/_flow_graph_join_impl.h

```diff
@@ -667,12 +667,12 @@
 
     template<typename InputTuple, typename OutputTuple>
     class join_node_FE<reserving, InputTuple, OutputTuple> : public reserving_forwarding_base {
-    public:
+    private:
         static const int N = std::tuple_size<OutputTuple>::value;
```

Comment:
Should there be a comment here saying why these are private?

Reply:
As discussed [above](https://github.com/oneapi-src/oneTBB/pull/868#discussion_r950049713), this parameter is part of the internal implementation details of `join_node_FE` so it should not be part of `oneapi::tbb::join_node` public API. 
Do you think we really need an extra comment in the code for this?

Reply:
🤷  I'm just trying to consider a future reader looking here without much context thinking "Why are `N`, `output_type`, `input_type`, and `base_node_type` private?" I'm thinking that would be clarified if they saw `// These parameters are private because they are part of the internal implementation details of join_node_FE: After all, a join node doesn't have a single input port.`

Just a thought.